POLICY.md

```diff
@@ -0,0 +1,40 @@
+
+## Polls
+
+A cornerstone of the way Pulsar interacts with the community is via Polls. These polls are usually created on Discord, but may in the future be expanded to other platforms as needed.
+
+A Poll is used to allow the community at large to easily and quickly contribute to a conversation and provide thier opinions on future development or goals of the Pulsar Project.
+
+* Whenever a decision will effect a large percentage of users/contributors, a change is substantal enough, or will drastically effect a small percentage of users/contributors then a poll must be made.
+* A Poll must be available for voting for at least 24 hours, to accommodate different timezones, and work schedules. Polls can be longer than this, but must never be shorter.
+* If a poll fails, then you must not recreate the exact same poll right afterwards. If the same idea has new merit or new validation at a later time, communicate with the community to determine if another poll is reasonable.
+
+## Repeat Questions
+
+Many times Pulsar throughout our various social platforms will receive repeated suggestions, and sometimes these suggestions are untenable for whatever various reasons. In these cases if we find ourselves repeatedly using the same answer, a discussion will happen at the Pulsar Admin Team level, on whether or not to add this conversation to our official [FAQ](https://github.com/pulsar-edit/.github/blob/main/FAQ.md) at which point, if agreed a PR will be made, and the community at large will be able to discuss it being added. At which point if it's added that now becomes the official stance and will be used in further discussion on the topic.
+
+## Pull Request Merging
+
+Obviously Pull Requests are an important part to software development.
+
+We get a lot of them, and there's not always enough of us to review everything in a timely manner. But below are some general guidelines.
+
+* Prior to merging there should be a best effort to allow two approvals of a PR.
+* A Pull Request should not be merged right away unless there is aduquate reason to do so. (e.g. This PR fixes a serious bug in the application, and is of a rather small change. This PR fixes a security issue.)
+
+## Donations
+
+We accept and encourage contributors to donate to the platform when they fiscally can. As these donations help Pulsar be great, and do what it needs to do.
+
+A donation through Pulsar will always be used for the greater good of the editor and it's ecosystem at large. But donators must understand that the use of funds will be dependent on the discretion of Pulsars Core Maintainers.
+
+But officially when you donate this money is now used for the Pulsar project. A donation, unless as part of a gift for donating or other specifically indicated event, does not allow a donator any extra say in how Pulsar will operate. A donation does not allow directed use of the funds, and does not allow control of the funds after they have been donated.
```

Comment:
```suggestion
Any money donated can and will be used used for the Pulsar project. A donation, unless as part of a gift for donating or other specifically indicated event, does not allow a donator any extra say in how the Pulsar team will operate. A donation does not allow directed use of the funds, nore does it allow control of the funds after they have been donated.
```

> A donation, unless as part of a gift for donating or other specifically indicated event,

Not entirely sure what this means?

Reply:
That's just there, since in the future if we decide donating a certain value gets you a free sticker or shirt or something.

Basically a caveat to an event like that which will be super clear on OC's end.

Reply:
Maybe "reward" as a word then? Basically unless we specifically said you get x reward for donating y... everything applies as usual.

Reply:
Yeah fair enough, I'll add that bit